bin/jruby.sh

```diff
@@ -505,11 +505,37 @@ fi
 
 # ----- Detect Java version and determine available features ------------------
 # shellcheck source=/dev/null
-java_version=$(. "$JAVA_HOME/release" && echo "${JAVA_VERSION-}")
+if [ -f "$JAVA_HOME/release" ]; then
+    java_version=$(. "$JAVA_HOME/release" && echo "${JAVA_VERSION-}")
+
+    # treat 1.8 as 8
+    case $java_version in
+        1.8 | 1.8.*) java_major=8 ;;
+        *)  # Split version out for integer comparisons
```

Comment:
Totally personal preference but I would put this comment above the case statement to group related code closer together

Reply:
Fixed this in a subsequent change, one comment line above the case now.